libkpod/mount.go

```diff
@@ -0,0 +1,42 @@
+package libkpod
+
+import (
+	"github.com/containers/storage"
+	"github.com/pkg/errors"
+)
+
+// Mount mounts a containers layer and returns the mountpoint
+func Mount(store storage.Store, cname string, options string) (string, error) {
+	rcstore, err := store.ContainerStore()
+	if err != nil {
+		return "", errors.Wrapf(err, "Can't find container store")
+	}
+	name, err := rcstore.Lookup(cname)
+	if err != nil {
+		return "", errors.Wrapf(err, "error finding container %q", name)
+	}
+
+	mountPoint, err := store.Mount(name, options)
+	if err != nil {
+		return "", errors.Wrapf(err, "error finding container %q", name)
+	}
+	return mountPoint, err
+}
+
+// Umount unmounts a containers layer
+func Umount(store storage.Store, cname string) error {
+	rcstore, err := store.ContainerStore()
```

Comment:
This bypasses locking and synchronization.  Use the `store` object's `Unmount` method instead.